### PR TITLE
Automerge

### PR DESCRIPTION
**Pull request by saeltz**
Removes labelling.
Adds Scala Steward as GitHub Action.
Adds merge step to PR validation for PRs opened by `moia-pipeline`.

### CODE REVIEW

**Comment by saeltz** [comment]
Merging seems to work, as long as the _github-user_ for Scala Steward is actually `moia-pipeline`.

**Comment by joroKr21** [comment]
But if we use the GitHub action how will the user be `moia-pipeline`?

File: .github/workflows/pr-validation.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
           path: |
             ~/.ivy2/cache
             ~/.sbt
+            ~/.cache/coursier/v1
```
Comment (joroKr21) [inline .github/workflows/pr-validation.yml:16]: 👍

File: .github/workflows/scala-steward.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# This workflow will launch at 09:00 every Monday
+on:
+  schedule:
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
```
Comment (joroKr21) [inline .github/workflows/scala-steward.yml:5]:
```suggestion
```

File: .github/workflows/scala-steward.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  scala-steward:
+    runs-on: ubuntu-latest
+    name: Launch Scala Steward
+    steps:
+      - name: Launch Scala Steward
+        uses: scala-steward-org/scala-steward-action@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          sign-commits: true
```
Comment (joroKr21) [inline .github/workflows/scala-steward.yml:16]:
```suggestion
          sign-commits: true
          scala-steward-version: 0.8.1
```

**Review by joroKr21 — COMMENTED** [review]
If we will be using the GitHub action then I guess we should remove the project from https://github.com/moia-dev/scala-steward

**Comment by joroKr21** [comment]
I guess `moia-pipeline` doesn't have an email so we might want to define `author-email` but I'm not sure to what 😁

File: .github/workflows/scala-steward.yml (hunk 2 above)
Comment (saeltz) [reply .github/workflows/scala-steward.yml:5]: According to the docs, this allows to trigger Scala Steward manually. I wanted to use this to verify if the username is indeed `moia-pipeline`.

File: .github/workflows/scala-steward.yml (hunk 3 above)
Comment (saeltz) [reply .github/workflows/scala-steward.yml:16]: Why do you want to fix the version?

File: .github/workflows/scala-steward.yml (hunk 2 above)
Comment (joroKr21) [reply .github/workflows/scala-steward.yml:5]: Oh wow, an empty config does that? 🙈

File: .github/workflows/scala-steward.yml (hunk 3 above)
Comment (joroKr21) [reply .github/workflows/scala-steward.yml:16]: Because the default is old. But whatever, I guess it doesn't matter.

**Review by joroKr21 — APPROVED** [review]

File: .github/workflows/scala-steward.yml (hunk 2 above)
Comment (saeltz) [reply .github/workflows/scala-steward.yml:5]: https://github.com/scala-steward-org/scala-steward-action#how-can-i-trigger-a-run

**Pull request merged**